WordPress/src/main/java/org/wordpress/android/ui/themes/ThemeBottomSheetFragment.kt

```diff
@@ -0,0 +1,78 @@
+package org.wordpress.android.ui.themes
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.View.INVISIBLE
+import android.view.View.VISIBLE
+import android.view.ViewGroup
+import androidx.annotation.NonNull
+import androidx.lifecycle.ViewModelProvider
+import com.google.android.material.bottomsheet.BottomSheetBehavior.STATE_EXPANDED
+import com.google.android.material.bottomsheet.BottomSheetBehavior.from
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+import org.wordpress.android.R
+import org.wordpress.android.WordPress
+import org.wordpress.android.databinding.ThemeActivationBottomSheetBinding
+import org.wordpress.android.viewmodel.themes.ThemesViewModel
+import org.wordpress.android.viewmodel.themes.ThemesViewModel.BottomSheetUIState.Selection.KeepCurrentHomepage
+import org.wordpress.android.viewmodel.themes.ThemesViewModel.BottomSheetUIState.Selection.UseThemeHomepage
+import javax.inject.Inject
+
+class ThemeBottomSheetFragment : BottomSheetDialogFragment() {
```

Comment:
This is more of a question than a change request. What led to the decision to rename the `ThemeActivationBottomSheetFragment` to `ThemeBottomSheetFragment`? My understanding is that this will be used for the activation functionality only.

Reply:
It was too long. And to me at least, it implied that there are action sheets in the themes module. In the future, if there are other bottom sheets in the themes we can use more descriptive names but until then it seemed overkill to me.